global/utils/constants.ts

```diff
@@ -37,3 +38,22 @@ const OVERTURE_DMS_DOCS_ROOT = 'https://overture.bio/documentation/dms/';
 export const DMS_HELP_URL = urlJoin(OVERTURE_DMS_DOCS_ROOT, 'user-guide');
 export const DMS_INSTALLATION_URL = urlJoin(OVERTURE_DMS_DOCS_ROOT, 'installation');
 export const DMS_EMAIL_SETTING_URL = urlJoin(DMS_INSTALLATION_URL, 'configuration/prereq/emails');
+
+// keycloak
+export const KEYCLOAK_URL_ISSUER = urlJoin(NEXT_PUBLIC_KEYCLOAK_HOST, 'realms', NEXT_PUBLIC_KEYCLOAK_REALM)
+export const KEYCLOAK_URL_TOKEN = urlJoin(KEYCLOAK_URL_ISSUER, 'protocol/openid-connect/token')
+export const KEYCLOAK_API_KEY_ENDPOINT = urlJoin(KEYCLOAK_URL_ISSUER, 'apikey/api_key')
+
+export const AUTH_PROVIDER = {
+    EGO: 'ego',
+    KEYCLOAK: 'keycloak'
+}
+
+export const INTERNAL_API_PROXY = {
+    ARRANGER: '/api/arranger',
+    PROTECTED_ARRANGER: '/api/protected/arranger',
+    PROTECTED_EGO_APIKEY_ENDPOINT: '/api/protected/ego/apikey',
+    PROTECTED_EGO_API_SCOPES_ENDPOINT: '/api/protected/ego/scopes',
+    PROTECTED_KEYCLOAK_APIKEY_ENDPOINT: '/api/protected/keycloak/apikey',
+    PROTECTED_KEYCLOAK_TOKEN_ENDPOINT: '/api/protected/keycloak/token'
+}
```

Comment:
Another hint that Prettier is not working for you. Let me know if you need a hand troubleshooting that
```suggestion
}

```